src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.h

```diff
@@ -128,6 +128,8 @@ class AppleseedRenderer
     virtual IOResult Save(ISave* isave) override;
     virtual IOResult Load(ILoad* iload) override;
 
+    static AppleseedInteractiveRender* g_interactive_renderer;
```

Comment:
I'm not sure to understand why we need a static data member here (it's a global variable and _will_ cause problems in multithreading scenarios).